release-to-candidate/README.md

```diff
@@ -24,12 +24,14 @@ jobs:
 
 ### Inputs
 
-| Key               | Description                                                                               | Required | Default     |
-| ----------------- | ----------------------------------------------------------------------------------------- | :------: | :---------- |
-| `architecture`    | The architecture for which to build the snap.                                             |    N     | `amd64`     |
-| `channel`         | The channel to release the snap to.                                                       |    N     | `candidate` |
-| `launchpad-token` | A token with permissions to create Launchpad remote builds.                               |    Y     |             |
-| `store-token`     | A token with permissions to upload and release to the specified channel in the Snap Store |    Y     |             |
+| Key               | Description                                                                               | Required | Default        |
+| ----------------- | ----------------------------------------------------------------------------------------- | :------: | :--------------|
+| `architecture`    | The architecture for which to build the snap.                                             |    N     | `amd64`        |
+| `channel`         | The channel to release the snap to.                                                       |    N     | `candidate`    |
+| `launchpad-token` | A token with permissions to create Launchpad remote builds.                               |    Y     |                |
+| `store-token`     | A token with permissions to upload and release to the specified channel in the Snap Store |    Y     |                |
+| `yaml-path`       | The path to the YAML file.                                                                |    N     |`Default Path`  |
```

Comment:
```suggestion
| `snapcraft-yaml-path`       | The path to the Snapcraft YAML file.                                                                |    N     |    |
```

Reply:
The default is computed if the input is left empty, but there is no default value set in the input specification itself.

Reply:
Oooo.. I thought these paths are default in snapcraft. Thanks for the info